.github/workflows/other-tests.yml

```diff
@@ -185,6 +185,9 @@ jobs:
             cd e2e/bug6933
             composer install
             ../../phpstan analyse -l 8 Module
+          - |
+            cd e2e/bug8610
+            ../../phpstan
```

Comment:
Put it in the `include` section lower, run it just once for a single PHP version.